package.json

```diff
@@ -12,7 +12,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "engines": {
-    "node": "12.x"
+    "node": ">=12.x"
```

Comment:
This unfortunately means it will affect the version used by the current live Heroku deployment too :/

Node.js 13.x isn't stable yet and Node.js 14 will be the next LTS which is planned to be released on April, see
https://nodejs.org/en/about/releases/ and https://github.com/nodejs/node/pull/32181

Reply:
I feel I need to clarify.

* I first attempted to install on a kitchen-counter Mac, an older retired laptop, that was running Node 8.  It wouldn't install.  After upgrading the kitchen Mac to Node 12 (latest LTS), everything works great.  So I added the commit to document Node 12 in the Install section of the Readme.

* I later installed on my production Mac which is running Node 13.  It installed with a warning (see above) ` Not compatible with your version of node/npm`.  This runs just fine on Node 13, which is why I made the second commit to modify `package.json.engines` because the hard-constraint on Node 12 is not appropriate.  This will be fine on Node 12+.

Reply:
Yeah, no worries, this should be fine :)

At first I thought the app was deployed on Heroku, which takes into consideration the engines version to choose which version of Node.js to use in production.

But this app is using Zeit Now. Now, I'm not sure if Zeit Now is using the same mechanism. If so, then this change means that the version of the deployed site will be 13.x, which is probably not intended for production.